src/lib/route-data/route-data.interface.ts

```diff
@@ -0,0 +1,3 @@
+export interface TCRouteDataInterface {
+    label: string;
```

Comment:
Hier muss auch noch ne LabelFunction möglich sein. Und in Terra ist das optional.. Müssen wir mal schauen ob wir das hier als Pflichtfeld setzen können.